src/main/java/uk/gov/hmcts/reform/idam/client/IdamClient.java

```diff
@@ -37,17 +45,38 @@
     private IdamApi idamApi;
     private OAuth2Configuration oauth2Configuration;
 
+    private static Cache<String, TokenResponse> accessTokenCache;
+    private final long refreshTokenBeforeExpiry;
+    private final boolean  enableCache;
```

Comment:
shouldn't it be `cacheEnabled` instead? or even `isCacheEnabled`?

Reply:
changed the name